packaging/cloudify-rest-service.spec

```diff
@@ -84,7 +84,7 @@ systemd-tmpfiles --create
 /etc/sudoers.d/cloudify-restservice
 /opt/restservice/NOTICE.txt
 /opt/restservice/set-manager-ssl.py*
-/usr/lib/systemd/system/cloudify-rest-service.service
+/usr/lib/systemd/system/cloudify-restservice.service
```

Comment:
Probably applies to the logrotate file as well?

Reply:
tbh I'd rather have them matching the package name as much as possible. This one has to be like this or else requires a bunch of changes elsewhere (cfy status etc.)
The name of the logrotate file is essentially arbitrary